src/ship-cache.js

```diff
@@ -10,17 +12,18 @@ export default class ShipCache {
    * @param {Object} cache instance of cache-manager
    * @param {String} namespace name of the namespace
    */
-  constructor(cache, namespace) {
```

Comment:
I liked the idea to have a namespace here. Allows us to bust cache for a specific namespace on a shared redis or memcached for example..